en/flying/package_delivery_mission.md

```diff
@@ -0,0 +1,84 @@
+# Package Delivery Mission Planning
```

Comment:
1. So a land waypoint involves landing, doing gripper, taking off again.
   - How do you stop the system from auto disarming? Or is the theory that the gripper will unlatch and the next waypoint be executed so quickly that the disarm won't happen? Do you need to tune anything to make this happen?
   - Did you give up on the idea of having something wait in place? i.e. a loiter waypoint on the ground, kicked off through the UI to go to next waypoint?
   - I assume that PX4 will just take off again to the new waypoint? i.e. no need for an explicit second takeoff waypoint?
   - Is that still true on a VTOL frame? They seem to like TAKEOFF waypoints.
2. An MC can hover, so it can drop a package too. See my note on the image below. 
3. If you're on a VTOL do you need to force a transition to MC first? Should we say that? 

I will need to subedit this a bit more for English, but I like it.

Reply:
For the images, for example, something like this is possible for the waypoint editor - highlighting the buttons you indicate should be selected
![image](https://user-images.githubusercontent.com/5368500/200742796-7e6576f0-990d-45ea-a5e0-84ed9d45739c.png)

Reply:
Definitely true, thanks for the tip!

Reply:
Impossible after the fact. But you could halve the height (as shown) and then zoom the mission. Obviously I can't fake zooming the mission, but you can see how you might do this.

![image](https://user-images.githubusercontent.com/5368500/203472595-3fea588a-a399-4b1b-ac1b-b90c165ead57.png)